### ISSUE TITLE
[NBug] fatal: bad revision '310'

### ISSUE THREAD

**Issue by JakobLettenbichler** [gitextensions/gitextensions#11103]
### Environment

- Git Extensions 4.0.2.16100
- Build 25100ec1f7c8da8f798613da74826038af81a50e
- Git 2.40.1.windows.1
- Microsoft Windows NT 10.0.19042.0
- .NET 6.0.13
- DPI 96dpi (no scaling)

### Issue description

Exit code: 128
Command: C:\Program Files\Git\bin\git.exe
Arguments: -c log.showSignature=false log -z --pretty=format:"%H%T%P%n%at%n%ct%n%aN%n%aE%n%cN%n%cE%n%B" --max-count=100000 --glob=refs/stas[h] 310 --
Working directory: C:\_dev\srcRabbit_qt4_3GDev\common\platform\

```
GitExtUtils.ExternalOperationException: fatal: bad revision '310'
 ---> System.Exception: fatal: bad revision '310'
   --- End of inner exception stack trace ---
   at void GitUI.RevisionGridControl.PerformRefreshRevisions(Func<RefsFilter, IReadOnlyList<IGitRef>> getRefs, bool forceRefresh)+() => { }
   at async Task GitUI.GitUIExtensions.InvokeAsync(Control control, Action action, CancellationToken token)
   at async void GitUI.TaskManager.FileAndForget(Task task, Func<Exception, bool> fileOnlyIf)+(?) => { }
```

I searched for "310" in the branches search field and pressing enter lead to the upper error

### Steps to reproduce

I searched for "310" in the branches search field and pressing enter lead to the upper error

**Comment by JakobLettenbichler** [comment]
Seems to be a duplicate from 

  * [![#10740](https://camo.githubusercontent.com/d0cd658f74f80f4718eee9f0baefbe49fbe06491244c6451d9e6e4de54a23ff2/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f25323331303734302532302535424e4275672535445f666174616c2533412532306261642532307265766973696f6e25323027313432343830272d73696d696c61726974792532303837253235253230253246253230636f6d6d656e747325323031382d7265642e737667)](10740)


therefore closing this issue.

**Comment by pmiossec** [comment]
> Seems to be a duplicate from

Yes and it has been fixed in v4.1 so you can update your version.

**Comment by JakobLettenbichler** [comment]
many thanks!
